modules/kubernetes-addons/kube-prometheus-stack/variables.tf

```diff
@@ -0,0 +1,28 @@
+variable "helm_config" {
+  type        = any
+  default     = {}
+  description = "Helm Config for kube-prometheus-stack"
+}
+
+# variable "manage_via_gitops" {
+#   type        = bool
+#   default     = false
+#   description = "[not implemented] Determines if the add-on should be managed via GitOps."
+# }
```

Comment:
May be its better to remove all the commented stuff. We can add this when we want to add a support to ArgoCD repo

Reply:
I think i finally got all the comments